docs/developer-docs/smart-contracts/call/overview.mdx

```diff
@@ -13,25 +13,40 @@ import { GlossaryTooltip } from "/src/components/Tooltip/GlossaryTooltip";
 
 <MarkdownChipRow labels={["Beginner", "Concept"]} />
 
-## Introduction
-
 To interact with a <GlossaryTooltip>canister</GlossaryTooltip>'s methods, there are two primary types of calls that can be used: update calls and query calls.
 
-Canisters implement methods that can be called via update or query calls. A query method can be called as both an update and a query, whereas update methods can be called only as an update.
-
 **Update calls** can make modifications to a canister's state, while **query calls** cannot.
 
-<div class="text--center">
-<img src="/img/docs/update_vs_query.png" alt="Update vs Query" width="400"/>
-</div>
+A query method can be called as both an update and a query, whereas update methods can be called only as an update.
+
+| Update calls | Query Calls |
+| :----------: | :------: |
+| Slow (1-2s) | Fast (200-400ms) |
+| Can modify state | Can't modify state |
+| Goes through consensus | Does not go through consensus |
+| Synchronous response | Synchronous response |
+| Executed on all nodes of a subnet | Executed on a single node |
+
 
 See the reference on [ingress messages](/docs/current/references/ingress-messages) for a more technical discussion of this topic.
 
 ## Update calls
 
-Update calls are executed on all nodes of a <GlossaryTooltip>subnet</GlossaryTooltip> since the result must go through consensus. This makes them slower compared to query calls. They are submitted asynchronously and answered synchronously.
+Update calls are executed on all nodes of a <GlossaryTooltip>subnet</GlossaryTooltip> since the result of the call must go through consensus. This makes them slower than query calls. They are submitted asynchronously and answered synchronously.
+
+### Making update calls
+
+To make a update call to a canister, use the [`dfx canister call`](/docs/current/developer-docs/developer-tools/cli-tools/cli-reference/dfx-canister#dfx-canister-call) command with the `--update` flag:
+
+
+- `dfx canister call --update <canister_name> <method_name>` : Make an update call to a canister deployed locally. The local replica must be running to create a canister locally. Start it with `dfx start --background`.
 
-### Example update call
+- `dfx canister call --update <canister_name> <method_name> --network=playground`: Make an update call to a canister deployed on the playground. Making update calls to canisters deployed on the playground is free, but canisters are temporary and will be removed after 20 minutes.
```

Comment:
Should we make it clear what's the playground and/or it's only for Motoko canisters?

Reply:
The playground is not just for Motoko canisters, it supports Rust as well.